requests/models.py

```diff
@@ -577,6 +577,10 @@ def prepare_hooks(self, hooks):
         for event in hooks:
             self.register_hook(event, hooks[event])
 
+    def send(self, session):
```

Comment:
i can peel that out if you're really against it — i think it's harmless though, and just smooths out the api a bit